mail_activity_team/models/calendar_event.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2019 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, _
+
+
+class CalendarEvent(models.Model):
+    _inherit = 'calendar.event'
+
+    def _get_default_team_id(self, user_id=None):
+        if not user_id:
+            user_id = self.env.uid
+        res_model = self.env.context.get('default_res_model')
+        model = self.env['ir.model'].search(
+            [('model', '=', res_model)], limit=1)
+        domain = [('member_ids', 'in', [user_id])]
+        if res_model:
```

Comment:
this check should go before ir.model search, no?

Reply:
True, I just changed it